dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10 AS builder
+RUN \ 
+    apt-get update && apt-get upgrade -q -y && \
+    apt-get install -y ca-certificates git node-gyp make
+WORKDIR /root/
+RUN git clone https://github.com/etclabscore/jade-explorer.git
+WORKDIR /root/jade-explorer/
+RUN \
+    npm install && \
+    npm run build    
+
+FROM httpd:2.4-alpine
```

Comment:
I have a local dockerfile going against `nginx:alpine` that ends up being `26mb` instead of `127mb`

Reply:
```
FROM node:10 AS builder
RUN \
    apt-get update && apt-get upgrade -q -y && \
    apt-get install -y ca-certificates git node-gyp make
WORKDIR /root/
RUN git clone https://github.com/etclabscore/jade-explorer.git
WORKDIR /root/jade-explorer/
RUN \
    npm install && \
    npm run build


# STEP 2 build a small nginx image with static website
FROM nginx:alpine

## Remove default nginx website
RUN rm -rf /usr/share/nginx/html/*

COPY --from=builder /root/jade-explorer/build/ /usr/share/nginx/html
EXPOSE 80
CMD ["nginx", "-g", "daemon off;"]

```

Reply:
ideally though we done do 

```
RUN git clone https://github.com/etclabscore/jade-explorer.git
```

should just do

```
COPY . /root/jade-explorer
```

or something

Reply:
Awesome thanks for this, just updated the dockerfile with your changes. :)